assets/js/atomic/blocks/product-elements/image/attributes.ts

```diff
@@ -1,4 +1,4 @@
-export const attributes = {
+export const blockAttributes: Record< string, Record< string, unknown > > = {
```

Comment:
Mm, why are we setting this type like this? Block attributes are a schema definition of the attributes accepted by a specific block. In the context of a generic function accepting any block, this would be correct. In this case, we know this shape should be respected (i.e. this would allow any string as a key), so we should instead set this as read-only and respect the shape as it is, in my opinion.

We could achieve this like so:

```
export const blockAttributes = {
 // ...
} as const;
```

Reply:
In https://github.com/woocommerce/woocommerce-blocks/pull/7567#discussion_r1012752200, @gigitux suggested using `type BlockAttributes = {...}` instead of `Record< string, Record< string, unknown > >`. Based on pdToLP-bQ-p2, I created an interface instead and followed this pattern here.